OpenRA.Mods.Common/AI/HackyAI.cs

```diff
@@ -344,10 +349,15 @@ public void Activate(Player p)
 				resourceTypeIndices.Set(tileset.GetTerrainIndex(t.TerrainType), true);
 		}
 
-		// DEPRECATED: Bot modules should queue orders directly.
+		void IBot.QueueOrder(Order order)
+		{
+			orders.Enqueue(order);
+		}
+
+		// DEPRECATED: Modules should use IBot.QueueOrder instead
```

Comment:
Do you need the deprecated mechanism to still exist?

Reply:
The idea is to fix the uses of this as logic is moved from HackyAI into their own modules.